packages/web/package.json

```diff
@@ -39,5 +42,30 @@
       "last 1 firefox version",
       "last 1 safari version"
     ]
+  },
+  "devDependencies": {
+    "@typescript-eslint/eslint-plugin": "^3.2.0",
+    "@typescript-eslint/parser": "^3.2.0",
+    "eslint": "^7.2.0",
+    "eslint-config-airbnb": "^18.1.0",
+    "eslint-plugin-import": "^2.21.2",
+    "eslint-plugin-jsx-a11y": "^6.2.3",
+    "eslint-plugin-react": "^7.20.0",
+    "husky": "^4.2.5",
+    "lint-staged": "^10.2.10"
+  },
+  "husky": {
+    "hooks": {
+      "pre-commit": "lint-staged",
+      "pre-push": "yarn ts:check && yarn test --watchAll=false"
+    }
+  },
+  "lint-staged": {
+    "*.{js,jsx,ts,tsx,css,md,json}": [
+      "prettier --write"
+    ],
+    "*.{jsx,js,ts,tsx}": [
+      "eslint --fix"
```

Comment:
Did you consider only using `eslint --fix` for `js/jsx/ts/tsx`? And not the `eslint --fix` on top of `prettier`?
Chances are thy might collide, trying to fix one another. Also on our project we completely replaced `prettier` for JS with a fine-tuned `eslint`, to avoid prettier being overly opinionated in some cases like line-length:


```
Prettier, on the other hand, strives to fit the most code into every line. With the print width set to 120, prettier may produce overly compact, or otherwise undesirable code.
```
https://prettier.io/docs/en/options.html#print-width
https://prettier.io/docs/en/rationale.html#print-width

This is a kind of dumb behaviour in my opinion, because you can't write a code like so for example:
```
do('something')
  .then('smth')
  .then('smth')
  .then('smth')
```

If these are short enough, they'll always be compacted to:
```
do('something').then('smth').then('smth').then('smth')
```
by prettier, even though this might not be the most readable option.

Team, what do you think?

Reply:
@dubbha  Do you propose to remove prettier?

Reply:
Actually yes, but let's leave it for now, and see how it becomes a problem.